CDC.WebNotes.Application/Services/NotesService.cs

```diff
@@ -68,5 +71,24 @@ public async Task AddNoteCheckListItem(int noteId, CreateNoteCheckListItemDto ch
 
             await _notesRepository.SaveChanges();
         }
+
+        public async Task AddAttachment(int noteId, AttachmentDto attachmentDto)
+        {
+            Note note = await _notesRepository.GetNote(noteId);
+            if (note == null)
+            {
+                return;
```

Comment:
OMG

Reply:
we should add some logic for such a situations or throw something like KeyNotFoundException
but here as I remember we throw it in Repository